### DISCUSSION TITLE
Multiple devices

### DISCUSSION THREAD

**Discussion by stevoh8494** [Tasshack/dreame-vacuum#604] [Q&A] [answered]
Hello, I have L10 Ultra configured in Hass and it works well. I have now a second robot, the mova m1. When I try to add it in HASS, system says that device is already configured. How can I configure the second robot? I have the last beta version. Many thanks

**Comment by KingDispo** [comment]
Same problem / question here. 2 Devices linked in the dreame account existis. Only one device is usable in home assistant.

**Reply by Tasshack** [reply]
Probably using older version of the integration that does not have support for your second device.

**Reply by KingDispo** [reply]
I am using v2.0.0b15. Do you think the problem is that the second device "dreame Mova M1" is not supported by this integration? :(

**Reply by Tasshack** [reply]
Mova M1 aka `dreame.vacuum.r2380` is in the supported devices list but for some reason that I cannot remember I have commented out the model from config flow.

You can remove "#" from model number and try adding your device again to see it will work or not?
https://github.com/Tasshack/dreame-vacuum/blob/dev/custom_components/dreame_vacuum/config_flow.py#L123

**Comment by Tasshack** [comment, marked as answer]
I have created an issue about this.
https://github.com/Tasshack/dreame-vacuum/issues/607
